Juice/Classes/zh-Hans.lproj/GeneralPreferencesViewController.strings

```diff
@@ -0,0 +1,24 @@
+
+/*
+Contributor : Zhilian DAI 
+*/
+/* Class = "NSTextFieldCell"; title = "Rescan For Scales:"; ObjectID = "0hr-m8-QTZ"; */
+"0hr-m8-QTZ.title" = "重新扫描Emoji配置区间：";
```

Comment:
Hey folks, so sorry for taking a while to get back to you both. the "Rescan for scales" string could equally means "Rescan for configuration files", I'd like to avoid the usage of emoji in that particular string because users can create a configuration that is not emoji based at all.